Source/HYPFormsCollectionViewDataSource.m

```diff
@@ -304,11 +303,27 @@ - (void)disable:(BOOL)disabled
     }
 
     for (NSString *fieldID in fields) {
-        BOOL shouldDisable = (![fieldID isEqualToString:@"blank"] && ![self.formsManager.disabledFieldsIDs containsObject:fieldID]);
+        HYPFormField *field = [fields valueForKey:fieldID];
+        BOOL shouldChangeState = (![self.formsManager.disabledFieldsIDs containsObject:fieldID]);
 
-        if (shouldDisable) {
-            HYPFormField *field = [fields valueForKey:fieldID];
+        if (disabled) {
+            field.disabled = YES;
+        } else if (shouldChangeState) {
             field.disabled = disabled;
+
+            if (field.targets.count > 0) {
+                [self processTargets:field.targets];
+            } else if (field.type == HYPFormFieldTypeSelect) {
+                BOOL hasFieldValue = (field.fieldValue && [field.fieldValue isKindOfClass:[HYPFieldValue class]]);
+                if (hasFieldValue) {
+                    HYPFieldValue *fieldValue = (HYPFieldValue *)field.fieldValue;
+                    if (fieldValue.targets.count > 0) {
+                        dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(0.1 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
```

Comment:
Had some issues with ```processTargets``` calls clashing with the UICollectionView so I delayed it to ensure that the first call had been processed. A very wonky an unsafe solution and I'm pretty confident that this can be removed.

Gonna look into it a bit and see if this still needed